app/src/components/Card/CardPreview/index.tsx

```diff
@@ -1,95 +1,93 @@
-          <div className='col-span-3 overflow-hidden'>
-            {!isEmpty(tasks)
-              ? tasks
-                  ?.slice(0, 3)
-                  ?.map((task) => <TaskPreview key={'prevtask-' + task.id} task={task} />)
-              : card?.description?.map((line, index) => (
-                  <div key={'description-' + line + index} className='text-sm'>
-                    {line}
-                  </div>
-                ))}
-          </div>
-          {tasks?.length > 4 ? (
-            <span className='ml-3 flex items-center h-full underline text-sm'>
-              +{tasks.length - 4}
-            </span>
-          ) : null}
-        </div>
-        {/* <div className='grid grid-cols-5 mt-5'> */}
-        <div className='flex mt-5 sm:max-w-[390px] md:max-w-[300px] lg:max-w-[450px] '>
-          {!isEmpty(imageUrls) ? (
-            <div className='flex col-span-2 '>
-              {imageUrls?.map((url, idx) => (
-                <Image key={'image-' + url + '-' + idx} className='w-12' src={url} alt='' />
+        <div className='text-lg font-semibold font-rubik not-italic mb-4'>{card.title}</div>
+        {/* Tasks Preview */}
+        <div className='flex-grow overflow-hidden'>
+          {tasks.length > 0 ? (
+            <div className='flex flex-col gap-3'>
+              {tasks.map((task) => (
+                <TaskPreview key={task.id} task={task} />
               ))}
-              {card?.files?.length > 5 ? (
-                <span className='ml-3 flex items-center h-full font underline text-sm ml-12'>
-                  +{card.files.length - 5}
+              {taskCountExcess > 0 && (
+                <span className='ml-3 flex items-center h-full underline text-sm'>
+                  +{taskCountExcess}
                 </span>
-              ) : null}
+              )}
+            </div>
+          ) : (
+            <div className='text-[#46434e] text-sm font-normal font-["Rubik"] leading-tight overflow-hidden'>
+              {card.description?.slice(0, 2).join(' ')}
             </div>
-          ) : null}
+          )}
+        </div>
+        {/* Images Preview */}
+        {imageUrls.length > 0 && (
+          <div className='mt-5 flex'>
+            {imageUrls.map((url, idx) => (
+              <Image key={url + idx} className='w-12' src={url} alt='' />
+            ))}
+            {card.files?.length > MAX_IMAGES && (
+              <span className='ml-3 flex items-center h-full font underline text-sm'>
+                +{card.files.length - MAX_IMAGES}
+              </span>
+            )}
+          </div>
+        )}
+        {/* User Avatars */}
+        <div className='mt-2 flex justify-start items-center gap-2'>
+          <AvatarGroup>
+            {(card.users as IUser[])?.map((user) => (
+              <Avatar key={user.id} src={user.avatarUrl} alt={user.name} radius='xl' size='sm' />
+            ))}
+          </AvatarGroup>
```

Comment:
```tsx
<div className='flex'>
    {
      card.users.map((user) => (
        <AvatarDun key={'preview-' + card.id + '-' + user.id} user={user} />
      ))
    }
</div>
```

Reply:
If margin is needed here - take a look here https://github.com/prspktvs/dun/blob/d1791944525a763d1894a0e69e0d6319eb89e150/app/src/components/Project/ProjectSettingsModal.tsx#L124